packages/core/src/js/feedback/FeedbackWidget.types.ts

```diff
@@ -171,7 +171,7 @@ export interface FeedbackCallbacks {
   /**
    * Callback when a screenshot is added
    */
-  onAddScreenshot?: (attachFile: (filename: string, fileUri: string) => void) => void;
+  onAddScreenshot?: (screenshotAdded: (uri: string) => void) => void;
```

Comment:
I think the present-tense verb was right since users will call this to add screenshot `addScreenshot('my-screenshot')`

````suggestion
  onAddScreenshot?: (addScreenshot: (uri: string) => void) => void;
````